elements/timeline-grid.html

```diff
@@ -2,14 +2,28 @@
 <link rel="import" href="timeline-element.html">
 <link rel="import" href="../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
 
-<dom-module id="timeline-grid">
+<dom-module id="timeline-grid" current-count="{{currentCount}}">
```

Comment:
Is this necessary? The current count is defined in the properties of the timeline-grid. I think polymer takes care of the rest